content/snippets/capacity-planning.md

```diff
@@ -0,0 +1,117 @@
+
+### Recommendation
+
+For best performance in enterprise production environments, Ripple recommends running `rippled` on bare metal with the following characteristics:
+
+- Operating System: Ubuntu 16.04+
+- CPU: Intel Xeon 3+ GHz processor with 4 cores and hyperthreading enabled
+- Disk: SSD
+- RAM:
+	- For testing: 8GB+
+	- For production: 32GB
+- Network: Enterprise data center network with a gigabit network interface on the host
+
+#### SSD Storage
+
+SSD storage should support several thousand of both read and write IOPS. The maximum reads and writes per second that Ripple engineers have observed are over 10,000 reads per second (in heavily-used public server clusters), and over 7,000 writes per second (in dedicated performance testing).
+
+#### CPU Utilization and Virtualization
+Ripple performance engineering has determined that bare metal servers achieve maximum throughput. However, it is likely that hypervisors cause minimal degradation in performance.
+
+#### Network
+
+Any enterprise or carrier-class data center should have substantial network bandwidth to support running `rippled` servers. The minimum requirements are roughly 2Mbps transmit and 2Mbps receive for current transaction volumes. However, these can burst up to 100MBps transmissions when serving historical ledger and transaction reports. When a `rippled` server initially starts up, it can burst to over 20Mbps receive.
+
+#### Storage
+
+Ripple recommends estimating storage sizing at roughly 12GB per day of data kept online with NuDB. RocksDB requires around 8GB per day. However, the data per day changes with activity in the network. You should provision extra capacity to prepare for future growth. Currently, a server with all XRP Ledger history requires 6.8TB (2018-03-01).
+
+<!-- {# ***TODO: Update the dated storage consideration above, as needed. ***#} -->
+<!-- {# ***TODO: DOC-1331 tracks: Create historic metrics that a user can use to derive what will be required. For ex, a chart with 1TB in 2014, 3TB in 2015, 7TB in 2018 ***#} -->
+
+#### Memory
+
+Memory requirements are mainly a function of the `node_size` configuration setting and the amount of client traffic retrieving historical data. As mentioned, production servers should maximize performance and set this parameter to `huge`.
+You can set the `node_size` parameter lower to use less memory, but you should only do this for testing. With a `node_size` of `medium`, a `rippled` server can be reasonably stable in a test Linux system with as little as 8GB of RAM.
+
+#### Amazon Web Servives
+
+Amazon Web Services (AWS) is a popular virtualized hosting environment. You can run `rippled`  in that environment. Ripple does not recommend using Elastic Block Storage (EBS), because the maximum number of IOPS (5,000) may not be able to keep up with the heaviest demands, but it is very expensive.
```

Comment:
> but it is very expensive.

Does that mean the preceding phrase should be "because the maximum number of IOPS (5,000) **may be able to** keep up with the heaviest demands"?

Reply:
It doesn't. EBS is expensive _and_ potentially insufficient.

A previous version of this sentence used "yet" instead of "but"—an interesting case where the two conjunctions aren't synonymous and "yet" has a meaning that conveys the intended meaning better than "but" does. That said, I think it's best to rewrite around the issue completely.

Here's my suggested rewrite:

> You can run `rippled`  in AWS, but Ripple does not recommend using Elastic Block Storage (EBS). Elastic Block Storage's maximum number of IOPS (5,000) is insufficient for `rippled`'s heaviest loads, despite being very expensive.